doc/plugins/limit-count.md

```diff
@@ -109,6 +110,41 @@ curl -i http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335
 }'
 ```
 
+If using redis-cluster:
+
+```shell
+curl -i http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "uri": "/index.html",
+    "plugins": {
+        "limit-count": {
+            "count": 2,
+            "time_window": 60,
+            "rejected_code": 503,
+            "key": "remote_addr",
+            "policy": "redis-cluster",
+            "redis_serv_list": [
+            	{
```

Comment:
avoid using the `tab` key, we always use spaces.

Reply:
how about this style? it seems simpler:

```
redis_serv_list: ["127.0.0.1:6373", "127.0.0.1:6374"]
```

Reply:
In the old version of the configuration, redis_host and redis_port were used.